### PR TITLE
Remove some biases and stereotypes from our word embeddings

### PR DESCRIPTION
**Pull request by rspeer**
Also includes code that uses a trie for faster OOV lookups, which I was working on at the same time.

### CODE REVIEW

File: conceptnet5/vectors/debias.py [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+    svc = svm.SVC(
+        verbose=False, random_state=0, max_iter=10000, class_weight='balanced',
+        probability=True, kernel='linear'
+    )
+    svc.fit(vecs, values)
+    return svc
+
+
+def de_bias_category(frame, category_examples, bias_examples):
```
Comment (jlowryduda) [inline conceptnet5/vectors/debias.py]: I think `de_bias_category`, `de_bias_binary`, and `two_class_svm` would benefit from a docstring.

File: conceptnet5/vectors/evaluation/bias.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+import pandas as pd
+import scipy
+
+from conceptnet5.vectors import standardized_uri, normalize_vec
+from conceptnet5.vectors.transforms import (
+    l2_normalize_rows, subtract_mean_vector
+)
+from conceptnet5.vectors.debias import (
+    FEMALE_WORDS, MALE_WORDS, PEOPLE_BY_BELIEF, PEOPLE_BY_ETHNICITY,
+    CULTURE_PREJUDICES, get_category_axis, get_vocabulary_vectors
```
Comment (jlowryduda) [inline conceptnet5/vectors/evaluation/bias.py]: CULTURE_PREJUDICES is not used in this file.

File: setup.py [hunk 3]
```diff
@@ -36,6 +36,6 @@
         ]
     },
     extras_require={
-        'vectors': ['numpy', 'scipy', 'statsmodels', 'tables', 'pandas', 'scikit-learn', 'mecab-python3', 'jieba']
+        'vectors': ['numpy', 'scipy', 'statsmodels', 'tables', 'pandas', 'scikit-learn', 'mecab-python3', 'jieba', 'marisa_trie']
     },
 )
```
Comment (jlowryduda) [inline setup.py]: Should we also include these packages on a Build process wiki page?

Comment (rspeer) [inline setup.py]: I've been dealing with the added dependency on wordfreq with jieba and mecab. marisa_trie should be an easy dependency, though, that just gets installed via setup.py.

File: conceptnet5/vectors/evaluation/bias.py (hunk 2 above)
Comment (rspeer) [inline conceptnet5/vectors/evaluation/bias.py]: It's used at the bottom, in `de_bias_frame`. It's not currently used separately from `SEX_PREJUDICES`, I suppose.

Comment (jlowryduda) [inline conceptnet5/vectors/evaluation/bias.py]: I think you may be looking at debias.py, not bias.py. With this comment, I was mostly wondering if this was intended that you don't measure `CULTURE_PREJUDICES`.

Comment (rspeer) [inline conceptnet5/vectors/evaluation/bias.py]: Aha, right.

I currently have something commented out that calculates `trained_ethnic_bias`, where it evaluates on `CULTURE_PREJUDICES`, which is exactly the correlation that it was trained to remove.

I found this useful as a test that bias removal was working, but not as a number that made any sense to compare between systems, so I commented it out. I guess I should come to a more solid conclusion about what to do with that measurement than just having it commented out.

**Pull request merged**